hermes_core/util/schema.py

```diff
@@ -1321,8 +1299,7 @@ def _get_generation_date(self, data):
         """
         Function to get the date that the CDF was generated.
         """
-        day = datetime.date.today()
-        return datetime.datetime(day.year, day.month, day.day)
+        return Time.now().strftime("%Y-%m-%d")
```

Comment:
Should further specify the date here in case we generate two CDF files in a single day?

Reply:
@ehsteve The previous implementation just included the date (no time) so I followed that paradigm. We would only have naming conflicts if the other portions of the name were also the same: instrument, start time, mode, data level... 
I do not think we'll meet these conditions where all aspects are the same, but if you think we will, I can update to include a timestamp in addition to the date.